[New Product] phpBB

Rudloff opened the pull request:
Unfortunately, I could not find a recent release policy.

Rudloff committed:
Add phpBB

marcwrobel commented on products/phpbb.md:
```diff
@@ -0,0 +1,26 @@
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: false
+releases:
```
https://github.com/phpbb/phpbb/tags could be used to automatically update latest patch versions using the following configuration:

```
-   git: https://github.com/phpbb/phpbb.git
    regex: ^release-(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)$
```

It's not supported yet, but we could use https://packagist.org/packages/phpbb/phpbb too.

captn3m0 replied:
packagist only has 3.0.2+, so git tags is better (for now).

marcwrobel commented on products/phpbb.md:
```diff
@@ -0,0 +1,26 @@
+    latest: "3.1.12"
+-   releaseCycle: "3.0"
+    eol: 2015-05-01
+    latest: "3.0.14"
```
It would be great to have the release date too. We could use tag dates (https://github.com/phpbb/phpbb/tags) as release dates.

marcwrobel commented on products/phpbb.md:
```diff
@@ -0,0 +1,26 @@
+
+---
+
+> [phpBB](https://www.phpbb.com/) is an open-source forum software written in PHP.
```
Even if a recent release policy could not be found it would be great to have an overview of what the phpBB release and support policy actually looks like. This overview should document answers to questions such as:

- Does phpBB follows semantic versioning ?
- Is there fixed release cycles or not ?
- What are the supported released ? Only the last X releases ?
- Is there multiple level of support (active, security...) ?

Rudloff replied:
It seems [it does follow semantic versioning](https://area51.phpbb.com/docs/dev/master/development/processes.html#target-versions).

I found [this archived page](https://web.archive.org/web/20190627185122/https://www.phpbb.com/about/release/) with more recent information than the post I originally used.
It actually differentiates between "end of maintenance" and "end of life" so I added an active maintenance column.

It seems there was a schedule at some point ("we plan to release a new minor (x.y) or major (x) update once every year around November/December", "EOL occurs to a branch 24 months after its first stable (3.x.0) release"), but now the latest minor release has been 3.3 since 2020.

Rudloff replied:
I just noticed the announced EOL date for 3.2 would have been before when 3.3 was actually released, so 3.2 was probably not really EOLed on that date.
However, I could not find a 3.2 EOL announcement :slightly_frowning_face:

marcwrobel replied:
Congrats finding [this link](https://web.archive.org/web/20190627185122/https://www.phpbb.com/about/release/) ! It seems outdated though, so it may be a good idea to ask on PhpBB support forum (https://www.phpbb.com/community/viewforum.php?f=551).

And if they do not have a release policy anymore we could something similar to:

> phpBB [follows semantic versioning](https://area51.phpbb.com/docs/dev/master/development/processes.html#target-versions). The support policy is not clearly defined, but it seems only the latest minor release sees active development and gets bug and security fixes.

marcwrobel reviewed (changes requested):
Hi @Rudloff, thank you for you contribution. I left you a few notes, do not hesitate to ask if you need some help.

Rudloff committed:
phpbb: add auto-update

Rudloff committed:
phpbb: add release dates

Rudloff committed:
fixup! phpbb: add release dates

Rudloff committed:
phpbb: use more recent information

From https://web.archive.org/web/20190627185122/https://www.phpbb.com/about/release/

Rudloff committed:
fixup! phpbb: use more recent information

Rudloff committed:
phpbb: add more info

captn3m0 committed:
Update phpbb.md

captn3m0 reviewed (approved):
LGTM.

marcwrobel committed:
normalize

marcwrobel reviewed (approved):
@Rudloff, sorry for taking so much time to have a look at this PR (thanks @captn3m0 for the "reminder" ;)). Just normalized a little bit the file and LGTM. Will merge.
